notifications.json

```diff
@@ -0,0 +1,478 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "version": "1.0",
+        "title": "Notification API",
+        "description": "Notification API"
+    },
+    "host": "e2effzjxvtgbmuilw-mock.stoplight-proxy.io",
+    "basePath": "/v1",
+    "schemes": [
+        "https"
+    ],
+    "paths": {
+        "/Messages/{id}": {
+            "parameters": [
+                {
+                    "name": "id",
+                    "in": "path",
+                    "required": true,
+                    "type": "string"
+                }
+            ],
+            "get": {
+                "operationId": "getMessage",
+                "summary": "Get Message",
+                "tags": [
+                    "Messages"
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Success: message exists.",
+                        "schema": {
+                            "allOf": [
+                                {
+                                    "$ref": "#/definitions/Message"
+                                }
+                            ],
+                            "description": ""
+                        }
+                    },
+                    "400": {
+                        "$ref": "#/responses/trait:apiErrors:400"
```

Comment:
We need descriptions of return codes at the request level, not in the trait. The meaning of a 404 changes from one endpoint to another.

For instance what's the meaning of 400 returned by this method? I don't see any way a GET for a specific notification could return a Bad Request